django/applications/catmaid/static/js/widgets/treenode-table.js

```diff
@@ -382,7 +382,8 @@
       if (event.which == 13) {
         // Filter with a regular expression
         this.filter_searchtag = $('#search_labels' + this.widgetID).val();
-        this.oTable.fnFilter(this.filter_searchtag, 2, true);
```

Comment:
Sure, will do next time - it was part of another commit way down in the history where I made some meaningful changes as well, and then I squashed all of them, so it would have been a pain to fish out.